params/chromium-params.md

```diff
@@ -75,3 +77,7 @@ Chromium's implementation assigns the following values:
 [allowed aggregatable budget per source value]: https://source.chromium.org/chromium/chromium/src/+/main:content/browser/attribution_reporting/attribution_config.h;l=97;drc=3733a639d724a4353463a872605119d11a1e4d37
 [Randomized aggregatable report delay]: https://wicg.github.io/attribution-reporting-api/#randomized-aggregatable-report-delay
 [randomized aggregatable report delay value]: https://source.chromium.org/chromium/chromium/src/+/main:content/browser/attribution_reporting/attribution_config.h;l=106;drc=3733a639d724a4353463a872605119d11a1e4d37
+[Max information gain for navigations]: https://wicg.github.io/attribution-reporting-api/#max-event-level-channel-capacity-per-source
+[max information gain for navigations value]: https://source.chromium.org/chromium/chromium/src/+/main:content/browser/attribution_reporting/attribution_config.h;l=113;drc=c8a629d7be311323904c523d33693af8f52d17e9
+[Max information gain for events]: https://wicg.github.io/attribution-reporting-api/#max-event-level-channel-capacity-per-source
+[max information gain for events value]: https://source.chromium.org/chromium/chromium/src/+/main:content/browser/attribution_reporting/attribution_config.h;l=114;drc=c8a629d7be311323904c523d33693af8f52d17e9
```

Comment:
This link is not correct, as it still shows the old value. This will have to be updated for the corresponding Chromium CL.

Reply:
Would you prefer @apasel422 to leave it as 1.58 for now and update when https://chromium-review.googlesource.com/c/chromium/src/+/4780891 lands?

Reply:
Either way. It's just weird to put the new value as the text of the link but then have it go to a specific commit line in Code Search with the old value.

Reply:
The CL landed so this should be OK now. I think we need to either remove the `drc` param or update it though.

Reply:
edited